flow/core/kernel/vehicle/traci.py

```diff
@@ -727,7 +727,7 @@ def _multi_lane_headways(self):
                 for lane in range(max_lanes):
                     edge_dict[edge][lane].sort(key=lambda x: x[1])
 
-        for veh_id in self.get_rl_ids():
+        for veh_id in self.get_ids():
```

Comment:
this is necessary for custom lane change models (that need to access lane leaders and followers)